bpdm-common/src/main/kotlin/org/eclipse/tractusx/bpdm/common/dto/BusinessPartnerType.kt

```diff
@@ -22,5 +22,6 @@ package org.eclipse.tractusx.bpdm.common.dto
 enum class BusinessPartnerType {
     LEGAL_ENTITY,
     SITE,
-    ADDRESS
+    ADDRESS,
+    NULL
```

Comment:
Rather than adding NULL as an enum value, why not making fields which are of type BusinessPartnerType nullable? So in the sharingstate entity the business partner type could nullable